geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -3219,15 +3219,30 @@ public Object get(Object key, Object aCallbackArgument, boolean generateCallback
     checkReadiness();
     checkForNoAccess();
     discoverJTA();
+    long start = startGet();
     boolean miss = true;
-
-    Object value = getDataView().findObject(getKeyInfo(key, aCallbackArgument), this,
-        true/* isCreate */, generateCallbacks, null /* no local value */, disableCopyOnRead,
-        preferCD, requestingClient, clientEvent, returnTombstones);
-    if (value != null && !Token.isInvalid(value)) {
-      miss = false;
+    try {
+      Object value = getDataView().findObject(getKeyInfo(key, aCallbackArgument), this,
```

Comment:
I don't see any need to change this method. It is only for PartitionedRegion so it does not need to call startGet and endGet.